tests/wfqueue.test.ts

```diff
@@ -793,7 +793,7 @@ class InterProcessWorkflow {
           ],
           {
             cwd: process.cwd(),
-            env: { ...process.env, DBOS__VMID: workerId },
+            env: { ...process.env, DBOS__VMID: workerId, DBOS__APPVERSION: globalParams.appVersion },
```

Comment:
Note, with the new dequeue behavior introduced in this PR, the worker processes *MUST* have the correct app version in order to dequeue.